### PR TITLE
[BACKLOG-35528] convert log4j.xml to log4j2.xml with file rename

### PR DESCRIPTION
**Pull request by ddiroma**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 26s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 4m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 3s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: assemblies/prd-ce/src/main/resources/resource/system/osgi/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                   filePattern="logs/pdi.%d{yyyy-MM-dd}.log">
+         <PatternLayout>
+            <Pattern>%d %-5p [%c] %m%n</Pattern>
+         </PatternLayout>
+         <Policies>
+            <TimeBasedTriggeringPolicy />
+         </Policies>
+         <DefaultRolloverStrategy />
+      </RollingFile>
+      <Console name="PENTAHOCONSOLE">
```
Comment (bennychow) [inline assemblies/prd-ce/src/main/resources/resource/system/osgi/log4j2.xml:14]: Missing Threshold filter set to INFO.  Isn't it weird that we see this set to different levels in different log4j.xml files =)

**Review by bennychow — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 3m 38s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/prd-ce
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Review by bennychow — APPROVED** [review]

**Pull request merged**